sphinx/source/docs/user_guide/webgl.rst

```diff
@@ -23,56 +25,75 @@ To enable WebGL in Bokeh, set the plot's ``output_backend`` property to
 
     p = figure(output_backend="webgl")
 
-Support
--------
+The WebGL output backend only supports a :ref:`subset of Bokeh's glyphs
+<userguide_webgl_supported_glyphs>`. When you enable WebGL, you are requesting
+that WebGL rendering is used if available. Glyphs that are not available in the
+WebGL output backend are rendered with the default canvas backend.
 
-Bokeh's WebGL support covers a subset of glyphs. This includes the :func:`~bokeh.plotting.figure.line`
-glyph, and most markers:
+.. _userguide_webgl_supported_glyphs:
 
-* :func:`~bokeh.plotting.figure.asterisk`
+Supported glyphs
+----------------
+
+Bokeh's WebGL support covers the following fixed-shape glyphs:
+
+* :func:`~bokeh.plotting.figure.block`
 
 * :func:`~bokeh.plotting.figure.circle`
 
-* :func:`~bokeh.plotting.figure.circle_cross`
+* :func:`~bokeh.plotting.figure.hbar`
 
-* :func:`~bokeh.plotting.figure.circle_x`
+* :func:`~bokeh.plotting.figure.hex_tile`
 
-* :func:`~bokeh.plotting.figure.cross`
+* :func:`~bokeh.plotting.figure.quad`
 
-* :func:`~bokeh.plotting.figure.diamond`
+* :func:`~bokeh.plotting.figure.rect`
 
-* :func:`~bokeh.plotting.figure.diamond_cross`
+* :func:`~bokeh.plotting.figure.scatter`
 
-* :func:`~bokeh.plotting.figure.hex`
+* :func:`~bokeh.plotting.figure.vbar`
 
-* :func:`~bokeh.plotting.figure.inverted_triangle`
+WebGL support for these fixed-shape glyphs covers the following properties:
 
-* :func:`~bokeh.plotting.figure.square`
+* all :ref:`fill properties <userguide_styling_fill_properties>`
+* all :ref:`line properties <userguide_styling_line_properties>` except for
+  dashed lines (which will be ignored)
+* all :ref:`hatch properties <userguide_styling_hatch_properties>` except for
+  hatch images using the ``hatch_extra`` property (which will be ignored).
 
-* :func:`~bokeh.plotting.figure.square_cross`
+There is also full WebGL support for :func:`~bokeh.plotting.figure.line`
+glyphs, although the appearance of dashed lines with round and square end caps
+needs improvement.
```

Comment:
```suggestion
glyphs, although the appearance of dashed lines with round and square end caps
may differ slightly from the default HTML canvas rendering.
```
Veers a little too much towards editorializing for docs

Reply:
Done.